projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -997,7 +997,6 @@ function get_paywall_blocks( $newsletter_access_level ) {
 		array(
 			'site_id'      => intval( \Jetpack_Options::get_option( 'id' ) ),
 			'redirect_url' => rawurlencode( get_current_url() ),
-			'v2'           => '',
```

Comment:
if we don't send this it will never enter `	if ( isset( $_GET['v2'] ) ) {` in `wp-content/mu-plugins/earn/api/v1/authentication/class-auth-endpoint.php:294`

Reply:
we need the distinction because `'https://' . $this->site_domain . '/wp-json/jetpack/v4/subscribers/auth'` we need to be sure we are dealing with a Jetpack site that has the 'https://' . $this->site_domain . '/wp-json/jetpack/v4/subscribers/auth' endpoint.

Reply:
Yeah that change was not meant to be here...